e2echart/e2e-chart-template.html

```diff
@@ -97,7 +97,17 @@ <h5 class="modal-title">Resource</h5>
         return false
     }
 
+    function isInterestingOrPathological(eventInterval) {
+        if (eventInterval.message.includes("pathological/true") || (eventInterval.message.includes("interesting/true"))) {
+            return true
+        }
+        return false
+    }
+
     function isPod(eventInterval) {
+        if (isInterestingOrPathological(eventInterval)) {
+            return false
+        }
```

Comment:
These 3 lines prevent events in `interesting-events` section from being duplicated in the `pods` section.